content/docs/onpremise/deploy.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Deployment On-premises"
```

Comment:
Change the directory name from `onpremise` to `on-premises`. There are two reasons for this suggestion. Firstly, see my comment below about the terms *premise* and *premises*. Secondly, it's better for SEO to separate words with hyphens in URLs.